packages/uix-host/src/port.ts

```diff
@@ -435,10 +435,7 @@ export class Port<GuestApi = unknown>
       try {
         methodCallee = this.getHostMethodCallee(address, privateMethods);
       } catch (e) {
-        this.logger.warn(
-          `Private method ${formatHostMethodAddress(address)} not found!`,
-          address
-        );
+        // ignore error, privateMethod gets called and works
```

Comment:
@nitinja , How can we be sure that a private method was actually called? On a line 441 below I see another if check statement
```
    if (!methodCallee) {
      methodCallee = this.getHostMethodCallee(address, this.hostApis);
    }
```
Probably this message might be useful, but needs to be moved into that if statement block below?

Reply:
Will not add any extra error handling in this case.